src/main/java/sirius/web/templates/NamedRowProcessor.java

```diff
@@ -28,5 +27,5 @@
      * @param lineNumber the line number which is currently being processed.
      * @param row        the data of the row / line to process
      */
-    void handleRow(int lineNumber, Map<String, Value> row);
+    void handleRow(int lineNumber, ListMultimap<String, Value> row);
```

Comment:
`ListMultimap` is already an interface. And I want to use Guava's MultiMap, which I cannot if I use an own interface